iati_dashboard/templates/codelist_publisher.html

```diff
@@ -0,0 +1,89 @@
+{% extends 'base.html' %}
+{% import '_partials/boxes.html' as boxes with context %}
+
+{% block page_header %}
+<h1>Publisher: {{ publisher.human_readable_name }}</h1>
+<h2>Codelist values used for <code class="iati-code">{{ element }}</code></h2>
+<p class="dashboard-page-heading__lead">(for files published to version {{ major_version }}.xx of the standard)</p>
+<p style="float:right">(<a href="{{ stats_url }}/current/aggregated-publisher/{{ publisher.short_name }}/codelist_values.json">This page in JSON format</a>)</p>
+<p>Values should be on the <code class="iati-code"><a href="{% if major_version=='2' %}http://iatistandard.org/codelists/{% else %}https://iatistandard.org/105/codelists/{% endif %}{{ codelist_mapping[major_version].get(element) }}/">{{ codelist_mapping[major_version].get(element) }}</a></code> codelist.</p>
```

Comment:
Should the version 2 links also be `https`?

Reply:
I've fixed this for this template. This was copy/pasted from another page, I see there's a bunch of http links that should be https, so I've made a new issue for that: https://github.com/IATI/IATI-Dashboard/issues/736 (this should be easy, but I've put off doing it now to avoid conflicts with your current work).